coral/src/app/features/login/index.ts

```diff
@@ -0,0 +1,4 @@
+import { LoginForm } from "src/app/features/login/LoginForm";
```

Comment:
We did have the barrel file in, yes. But I would agree to remove it. I still like the explicit exporting etc. but on the other hand I think more and more maybe we should rather avoid barrel files (maybe altogether?) and see if we find an alternative to be explicit about the api of a directory/section? 🤔 

The drawbacks with barrel file because potential issues with tree shaking, performance impact in dev mode but mostly testing, but also introducing potential site effects in the code are there and it can be hard to avoid them, especially if a project grows. 

One example how we could handle in the `feature` directory could be:

```
├── features
│   └── login/
│       ├── components
│       ├── SmallComponetForLogin.tsx
│       ├── hooks
│        │       ├── useLoginUser.tsx
│       ├── ...
│       └── Login.tsx

```

So basically having only one react component on root and this is the only one that is imported from the outside. 

**Drawback**
- may create unnecessary directories (e.g. having a `hook` directory with one hook in it
- we can't apply a common eslint rule for that, probably would have to be custom, which is of course way less convenient 

Maybe something we should revisit soon and think about how we want to do that?

I'll remove the barrel file (and the related example in the docs) in this PR, more changes can be done in a different one!

Reply:
Agreed! Lets put this in and as we have more feature and imports, we can revisit the structure.

Reply:
Sounds good! I'll make an issue in the backlog!